### ISSUE TITLE
HAPI-2223: Timer already cancelled

### ISSUE THREAD

**Issue by davpatrik** [hapifhir/hapi-fhir#5650]
**Describe the bug**
Error 500 when calling any endpoint in baseR4 and baseR5
`HAPI-2223: Timer already cancelled.; nested exception is java.lang.IllegalStateException: Timer already cancelled.`

**To Reproduce**
1. Call `GET /PractitionerRole` endpoint in baseR4 swagger
<img width="1793" alt="image" src="https://github.com/hapifhir/hapi-fhir/assets/51869286/80ddac2e-6902-455b-ae50-2d9178c1ff79">

2. Or, call `GET /PractitionerRole` endpoint in baseR5 swagger
<img width="1458" alt="image" src="https://github.com/hapifhir/hapi-fhir/assets/51869286/c3e4ac80-3277-4d42-9942-bb9ffc679dc6">

**Expected behavior**
To get a 200 answer

**Environment (please complete the following information):**
 - HAPI FHIR Version R4 and R5
 - Browser : chrome

**Comment by Vinodh324** [comment]
yeah, this server was working before 2 days and now its not, When can we expect the same to be fixed??

**Comment by jcalcedo** [comment]
It seems to be back running

**Comment by yogeet-mulik** [comment]
The server is down again

**Comment by trishan9** [comment]
Server is down again

**Comment by fsccn** [comment]
And down again

**Comment by cdimas1** [comment]
Response request always giving timer alrady cancelled https://prnt.sc/9be3t3OPYiwx  example GET https://hapi.fhir.org/baseR4/Flag/10985804

**Comment by monami44** [comment]
seems to be down again

**Comment by LucasJandrey5** [comment]
Server is down again
